src/css/custom.css

```diff
@@ -194,17 +275,20 @@ article a:focus {
 	text-decoration: none;
 }
 
-/* ian: Fix for dark mode later
-.pagination-nav__link:hover .pagination-nav__sublabel {
-  color: #2e54ea;
-}
-*/
-
 /* Left drawer menu */
+@media (min-width: 997px) {
+	.sidebarViewport_node_modules--pnpm-\@docusaurus\+theme-classic\@2-4-3_react-dom\@17-0-2_react\@17-0-2_typescript\@5-2-2-node_modules-\@docusaurus-theme-classic-lib-theme-DocPage-Layout-Sidebar-styles-module {
+		top: 1rem !important;
+	}
 
-.theme-doc-sidebar-menu {
-	padding-top: var(--ifm-menu-link-padding-horizontal);
-	padding-bottom: var(--ifm-menu-link-padding-horizontal);
+	.sidebar_node_modules--pnpm-\@docusaurus\+theme-classic\@2-4-3_react-dom\@17-0-2_react\@17-0-2_typescript\@5-2-2-node_modules-\@docusaurus-theme-classic-lib-theme-DocSidebar-Desktop-styles-module {
```

Comment:
This compiles as `.sidebarViewport_uxQu` 🤔

Reply:
div[class*="sidebarViewport"] should work (classname contains sidebarViewport)